src/component/term/TermMetadataEdit.tsx

```diff
@@ -150,6 +151,12 @@ export class TermMetadataEdit extends React.Component<
     this.setState({ parentTerms });
   };
 
+  public onExactMatchesChange = (exactMatchTerms?: Term[]) => {
+    this.setState({
+      exactMatchTerms: exactMatchTerms?.map((e) => e as TermInfo),
```

Comment:
Ok, now I probably see why there was the transformation routine for exactMatch attribute in `Term.toJsonLd`. In that case, I think the selected terms should be mapped to `TermInfo` here - i.e., create new objects with just the subset of attributes corresponding to `TermInfo`.